src/components/Registration/RegistrationStepOne/RegistrationStepOne.tsx

```diff
@@ -38,6 +39,12 @@ export const RegistrationStepOne = withRouter(
      */
     const { t } = useTranslation();
 
+    const isAdministrationAlreadyRegistered =
```

Comment:
since this appears twice, I'd refactor it to some utility function

Reply:
@gunzip passing it down as prop from registration container to registration step one is ok too?

Reply:
it should be ok, try :)